src/ape/managers/project/types.py

```diff
@@ -93,7 +93,7 @@ def create_manifest(
             cached_source_reference_paths = {
                 source_id: [
                     self.contracts_folder.joinpath(Path(s))
-                    for s in getattr(source, "references", [])
+                    for s in getattr(source, "references", []) or []
```

Comment:
isn't this what the 2nd argument in `getattr` does?
https://docs.python.org/3/library/functions.html#getattr

Reply:
Oh I see, maybe this is clearer
```suggestion
                    for s in getattr(source, "references", None) or []
```

Reply:
It's getting the attribute for `references` but the actual value is None 
![telegram-cloud-photo-size-1-4918042827766737427-y](https://user-images.githubusercontent.com/32708219/169573280-8b6aff37-28e7-43aa-a61a-7586c35efec4.jpg)

Reply:
tbh I'd eventually like to clean up `ethpm-types` when we eventually break from the standard, it would make the schema way cleaner